ProjectLibertyLabs/siwf
# Update Docs Theme

# Problem

Updating theme colors to the white/navy and mdbook to v0.4.42+

## Steps to Verify:

1. cargo install mdbook@0.4.42
1. cd docs
1. mdbook serve

Want to also verify the generator?
1. cd tools/signed-request-generator
2. npm i
3. npm run build
4. cd ../..
5. rm -Rf docs/src/Generator && cp -a tools/signed-request-generator/build docs/src/Generator

Also did an npm audit fix on the library and generator tool

## Screenshots (optional):

Light (New is on top):
![image](https://github.com/user-attachments/assets/f1819859-40d8-42b5-a86b-a72e2378adc3)



Dark (New is on top):

![image](https://github.com/user-attachments/assets/d59696b3-426f-471e-a63a-69d2889bfcc6)


Generator form (new on top):
![image](https://github.com/user-attachments/assets/733b64e1-edca-49b8-936d-4e27bd917fd7)
![image](https://github.com/user-attachments/assets/107b32dc-962f-4588-bd82-9c8b6d21c6eb)
